src/asset/style/layout/_base.scss

```diff
@@ -0,0 +1,33 @@
+html {
+	font-size: 62.5%;
```

Comment:
Shouldn't we set this to a hardcoded 10px instead of 62.5%? I usually use rem for width and height of elements and if the user changes it's default browser font size to anything else then 16px (for accessibility reasons) it could break your layout!